src/Monolog/Handler/FingersCrossedHandler.php

```diff
@@ -80,6 +80,29 @@ public function isHandling(array $record)
     }
 
     /**
+     * Manually activate this logger regardless of the activation strategy
+     */
+    public function activate()
+    {
+        if ($this->stopBuffering) {
+            $this->buffering = false;
+        }
+        if (!$this->handler instanceof HandlerInterface) {
+            $record = end($this->buffer);
```

Comment:
`$record = end($this->buffer) ?: null;` could replace those 4 lines but no biggie.